### PR TITLE
[SMALLFIX] Standardized duplicate error messages in master

### PR DESCRIPTION
**Pull request by manugoyal**
We put exception messages that are used in numerous places in the ExceptionMessage class. Other remaining error messages in the master code are not duplicated anywhere, so they didn't need to be put into ExceptionMessage

### CODE REVIEW

**Comment by manugoyal** [comment]
Jenkins, test this please

**Comment by apc999** [comment]
Thanks @manugoyal . Could you either add JIRA number or prefix the PR title with "[SMALLFIX]" to be consistent, and also could you resolve the merge conflicts?

**Comment by manugoyal** [comment]
Jenkins, test this please

File: servers/src/main/java/tachyon/master/block/BlockMaster.java [hunk 1]
```diff
@@ -42,7 +42,8 @@
 import tachyon.StorageDirId;
 import tachyon.StorageLevelAlias;
 import tachyon.conf.TachyonConf;
-import tachyon.IndexedSet;
+import tachyon.exception.ExceptionMessage;
+import tachyon.master.IndexedSet;
```
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/block/BlockMaster.java]: should this be tachyon.IndexedSet?

**Comment by jsimsa** [comment]
Left a comment for you that should fix the compilation failures. Other than that, it looks good to me.

**Comment by jsimsa** [comment]
LGTM +1

File: common/src/main/java/tachyon/exception/ExceptionMessage.java [hunk 2]
```diff
@@ -57,6 +57,13 @@
   TEMP_BLOCK_ID_COMMITTED("Temp blockId {0} is not available, because it is already committed"),
   TEMP_BLOCK_ID_EXISTS("Temp blockId {0} is not available, because it already exists"),
 
+  // file system master
+  UNEXPECETD_JOURNAL_ENTRY("Unexpected entry in journal: {0}"),
+  FILEID_MUST_BE_FILE("File id {0} must be a file"),
+  JOURNAL_WRITE_AFTER_CLOSE("Cannot write entry after closing the stream"),
+  RAW_TABLE_ID_DOES_NOT_EXIST("Raw table with id {0} does not exist"),
+  UNKNOWN_ENTRY_TYPE("Unknown entry type: {0}"),
```
Comment (apc999) [inline common/src/main/java/tachyon/exception/ExceptionMessage.java]: maybe `UNKNOWN_JOURNAL_ENTRY_TYPE` to be more specific in this name?

**Comment by apc999** [comment]
Thanks @manugoyal . the patch looks good in general, with one minor comment left.

File: common/src/main/java/tachyon/exception/ExceptionMessage.java (hunk 2 above)
Comment (manugoyal) [inline common/src/main/java/tachyon/exception/ExceptionMessage.java]: Actually it's used once for journal entries, and once for raw table entries, so I think I'll just remove this and make it specific in those two places

Comment (apc999) [inline common/src/main/java/tachyon/exception/ExceptionMessage.java]: sounds good.

**Comment by apc999** [comment]
+1 LGTM

**Pull request merged**